javascript/javascript1/week1/eznamey.js

```diff
@@ -0,0 +1,7 @@
+var firstWords= ["Easy", "Awesome", "Corporate", "Build", "Bakers", "Flow", "Sun", "Cloud" , "InterV","Cantor"];
+var secondWords = ["Blosum", "Philler","Beta", "Delta", "Data", "Level","Brand","Selection","Sonic","Collaboration"];
+//var startupName = firstWords[2] + secondWords[3]; This is manual option/////
+const randomNumber = Math.floor(Math.random() * 10) + 0;
```

Comment:
is '+0' needed?

Reply:
It will work without +0. 

I gave it to start from 0 to 10 .